src/Core/Services/Implementations/UserService.cs

```diff
@@ -561,10 +561,13 @@ public async Task InitiateEmailChangeAsync(User user, string newEmail)
             return result;
         }
 
+        var now = DateTime.UtcNow;
+
         user.Key = key;
         user.Email = newEmail;
         user.EmailVerified = true;
-        user.RevisionDate = user.AccountRevisionDate = DateTime.UtcNow;
+        user.RevisionDate = user.AccountRevisionDate = now;
```

Comment:
❓ Combine? I like separation personally. There's a few of these so they should be one line or multiple, each time.

Reply:
I can't combine the new `Last*Date` fields since they are nullable, so gives a type mismatch when you try to oneline it.